src/v/cluster/topics_frontend.cc

```diff
@@ -81,6 +82,13 @@ std::vector<std::string_view> get_enterprise_features(
     if (cfg.is_schema_id_validation_enabled()) {
         features.emplace_back("schema ID validation");
     }
+    if (const auto leaders_pref = cfg.cfg.properties.leaders_preference;
```

Comment:
nitpick: This doesn't need to be a copy.

Reply:
you're right. There is a vector in there.